test/language/english-transforms.test.js

```diff
@@ -26,16 +26,123 @@ const tests = [
         category: 'nouns',
         valid: true,
         tests: [
-            {term: 'cat', source: 'cats',  rule: 'ns', reasons: ['plural']}
+            {term: 'cat', source: 'cats',  rule: 'ns', reasons: ['plural']},
+            {term: 'cat', source: 'cat\'s',  rule: 'ns', reasons: ['possessive']},
```

Comment:
I'd rather split these tests by rules (e.g. `plural` or `possessive`) rather than source terms. I think it makes it easier to debug when tests break. What do you think @StefanVukovic99?

Reply:
![image](https://github.com/themoeway/yomitan/assets/24891609/5c6eee8d-d78f-4005-b335-7d84c922dddd)
The error message pretty much takes you to the individual rule, and these tests (rather than tests for every language) will probably break only if you tweak the english-transforms.js file. In which case, you also know what you changed so you can pinpoint. 

There's a case for both ways ( I think i went with this approach because the japanese file uses it), so I'd not require any changes here.

Reply:
I find it easier to identify coverage gaps in conjugation if it's separated by conjugation. Kind of an organizational nitpick